compute_tools/src/bin/compute_ctl.rs

```diff
@@ -84,6 +84,28 @@ fn main() -> Result<()> {
         }
     };
 
+    // Extract OpenTelemetry context for the startup actions from the spec.
+    //
+    // This is used to propagate the context for the 'start_compute' operation
+    // from the neon control plane. This allows linking together the wider
+    // 'start_compute' operation that creates the compute container, with the
```

Comment:
Stray "with the".

Reply:
Fixed, and improved the comment a little in general.